__tests__/app.test.js

```diff
@@ -324,4 +324,35 @@ describe("GET /api/users", () => {
       })
     })
   })
+})
+describe("GET /api/articles?topicquery", () => {
+  test("200 - sends array of article objects with requested topic", () => {
+    return request(app)
+    .get("/api/articles?topic=cats")
+    .expect(200)
+    .then(({body}) => {
+      const {articles} = body
+      expect(articles.length).toBe(1)
+      articles.forEach((article) => {
+        expect(article.topic).toBe("cats")
```

Comment:
This is nice and clean, but I am inclined to say it might be more secure to test that all the other properties are still present that we would expect in the article object too, so we know that the use of the query isn't somehow returning an object with keys that are missing